packages/create-svelte/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
what was this file added for? just for VS Code or something?

Reply:
Yes, similar to the other packages which also only have it for type checking